shared/actions/git.js

```diff
@@ -156,7 +156,7 @@ const _onTabChange = (action: RouteTreeTypes.SwitchTo) => {
 }
 
 function* _handleIncomingGregor(action: GitGen.HandleIncomingGregorPayload): Saga.SagaGenerator<any, any> {
-  const msgs = action.payload.messages.map(msg => JSON.parse(msg.body))
+  const msgs = action.payload.messages.map(msg => JSON.parse(msg.body.toString()))
```

Comment:
I think this worked before because the buffer is essentially casting to a string implicitly using toString() but this thing really is a buffer